build/css-selector-generator.js

```diff
@@ -4,7 +4,14 @@
 
   CssSelectorGenerator = (function() {
     CssSelectorGenerator.prototype.default_options = {
-      selectors: ['id', 'class', 'tag', 'nthchild']
+      selectors: ['id', 'class', 'tag', 'nthchild'],
+      prefix_tag: false,
+      log: false,
+      attribute_blacklist: [],
+      attribute_whitelist: [],
+      quote_attribute_when_needed: false,
```

Comment:
Why not just surrounding attribute values with quotes in all cases?

Reply:
To ensure that existing testcases do not fail :). Call is yours. And the CSS selector will increase by two bytes and I thought you wanted to generate the smallest possible selector.

Reply:
OK, I see :)